W-11280756 sap concur access token step (#1978)
mulesoft/docs-connectors: sap-concur/4.0/modules/ROOT/pages/index.adoc

|Authorization| A Concur access token.
|===
+

> . Create an access token for your Concur instance. Creating an access token for your Concur instance by specifying `+https://www.concursolutions.com:443/net2/oauth2/accesstoken.ashx+` in Postman with the GET method is https://developer.concur.com/api-reference/authentication/authorization-pre-2017.html[deprecated]. To create an access token, use one of the following authentication methods:

Reviewer: ```suggestion
. Create an access token for your Concur instance. To do this, specify: `+https://www.concursolutions.com:443/net2/oauth2/accesstoken.ashx+` in Postman with the GET method is https://developer.concur.com/api-reference/authentication/authorization-pre-2017.html[deprecated]. 
+
To create an access token, use one of the following authentication methods:
```

Reviewer: "with the GET method is" does not make sense. 
Do you mean "using the GET method ..." or "when the GET method is..."?

Reviewer: Also, I am confused by how the concursolutions.com URL is part of the GET method. Is it in the body? 

What does it mean that the GET method is deprecated? This is what shows in the Preview.

The GET method should be in back ticks.

* https://developer.concur.com/api-reference/authentication/apidoc.html#auth_grant[Authorization grant]
* https://developer.concur.com/api-reference/authentication/apidoc.html#password_grant[Password grant]
* https://developer.concur.com/api-reference/authentication/apidoc.html#client_credentials[Client Credentials grant]
* https://developer.concur.com/api-reference/authentication/apidoc.html#otp_grant[One Time Password grant]

> . After obtaining the access token for your Concur instance, use the access token in the configuration for SAP Concur Connector. The following screenshot shows an example of obtaining an access token using Password grant:

Reviewer: ```suggestion
. After obtaining the access token for your Concur instance, use the access token in the configuration for SAP Concur Connector. The following screenshot shows an example of obtaining an access token using the Password grant:
```

Reviewer: I know using "the" sometimes goes against Kim's advice, but I think this sounds funny when you don't use it.
